avm/res/recovery-services/vault/main.bicep

```diff
@@ -63,6 +63,15 @@ param securitySettings object = {}
 ])
 param publicNetworkAccess string = 'Disabled'
 
+@description('Optional. The encryption settings for the vault.')
+param encryption encryptionType
```

Comment:
Hey @alexanderojala,
this should (unless it's not totally different) implement the AVM spec for CMK  defined [here](https://azure.github.io/Azure-Verified-Modules/specs/shared/interfaces/#customer-managed-keys).

If you need help in this regard please let me know. The background of this requirement/spec is to try and keep the user experience when deploying CMK as consistent as possible across resource modules.